_posts/2017-08-29-Making-Artsy-Editorial-Accessible.md

```diff
@@ -0,0 +1,56 @@
+---
+layout: post
+title: Making Artsy Editorial Accessible
+date: 2017-07-06
+categories: []
+author: eve
+---
+
+During Artsy's recent 2017 Hackathon we tackled making all our our editorial content accessible. The idea was hatched at  Berlin JSConf this spring, where [Laura Carvajal gave a talk](https://youtu.be/H4FzW9oFObs) following the *Financial Times*' experience implementing better accessibility requirements, and how they built these considerations into their testing process.
```

Comment:
Could be worth linking "accessible" in the first sentence to https://en.wikipedia.org/wiki/Accessibility or somewhere, in case someone needs more context.

Reply:
Good call!